### ISSUE TITLE
Multiple tracking id support

### ISSUE THREAD

**Issue by ace-eddy** [react-ga/react-ga#102]
It does not support multiple tracking id this time.

import ReactGA from 'react-ga'
import ReactGATCK from 'react-ga'

ReactGA.initialize('UA-xxxxxxxx-1')
ReactGATwo.initialize('UA-xxxxxxxx-1')

function logPageView() {
  ReactGA.set({ page: window.location.pathname })
  ReactGA.pageview(window.location.pathname)
  ReactGATwo.set({ page: window.location.pathname })
  ReactGATwo.pageview(window.location.pathname)
}

**Comment by laurenskling** [comment]
I would like this too :)

**Comment by johhansantana** [comment]
I just did this and it's working
```javascript
    ReactGA.initialize('code1');
    ReactGA.initialize('code2');
    ReactGA.set({ page: window.location.pathname + window.location.search });
    ReactGA.pageview(window.location.pathname + window.location.search);
```

**Comment by SimeonC** [comment]
Closed via #145
